_includes/configuration/inline.md

```diff
@@ -23,6 +21,28 @@ tinymce.init({
   inline: true
 });
 ```
+
 > **Note**: {{site.productname}} in inline mode will not initialize on the following elements: `area`, `base`, `basefont`, `br`, `col`, `frame`, `hr`, `img`, `input`, `isindex`, `link`, `meta`, `param`, `embed`, `source`, `wbr`, `track`, `colgroup`, `option`, `table`, `tbody`, `tfoot`, `thead`, `tr`, `th`, `td`, `script`, `noscript`, `style`, `textarea`, `video`, `audio`, `iframe`, `object`, and `menu`.
 
 For more information on the differences between the editing modes, please see this page [here]({{ site.baseurl }}/general-configuration-guide/use-tinymce-inline/).
+
+
+### Inline editing and complex stylesheets
+
+When using {{site.productname}} in the inline mode, the editor inherits the CSS stylesheet from the page it is embedded in.
+
+If the editor is used in inline mode, care should be taken when using styling that depends on structures within the editor. For example, if there's a class like this:
```

Comment:
Doesn't this apply to all content CSS, not just inline mode? As the same would be true if you used this content CSS in iframe mode.